### PR TITLE
Use 'JSZipMock' instead of 'JSZip' to reduce time to run tests

### PR DESCRIPTION
**Pull request by IgnatovDan**

### CODE REVIEW

File: testing/helpers/jszipMock.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class Folder {
+    constructor() {
+        this.children = {};
```
Comment (jsdmitry) [inline testing/helpers/jszipMock.js:3]: Maybe need rename "children" to "files"

File: testing/helpers/jszipMock.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        let folder = this.children[name];
+        if(!folder) {
+            folder = new Folder();
+            this.children[name] = folder;
+        }
+        return folder;
+    }
+}
+
+export default class JSZipMock extends Folder {
```
Comment (jsdmitry) [inline testing/helpers/jszipMock.js:26]: Why use extend from the Folder class? Use the "Folder" as "JSZipMock".

File: testing/tests/DevExpress.ui.widgets.dataGrid/dataGrid.export.tests.js [hunk 3]
```diff
@@ -4,10 +4,19 @@ import "ui/data_grid/ui.data_grid";
...
 import { excel as excelCreator } from "client_exporter";
+import excel_creator from "client_exporter/excel_creator";
+import JSZipMock from "../../helpers/jszipMock.js";
+
+const oldJSZip = excel_creator.ExcelCreator.JSZip;
 
 QUnit.testStart(function() {
     var markup = '<div id="dataGrid"></div>';
     $("#qunit-fixture").html(markup);
+    excel_creator.ExcelCreator.JSZip = JSZipMock;
```
Comment (jsdmitry) [inline testing/tests/DevExpress.ui.widgets.dataGrid/dataGrid.export.tests.js]: Use this code inside the "beforeEach" and "afterEach" methods of module settings. For example, please see the \testing\tests\DevExpress.ui.widgets\accordion.tests.js

**Review by jsdmitry — CHANGES_REQUESTED** [review]
Move "excel_creator.ExcelCreator.JSZip = JSZipMock" to 
the "beforeEach" method of module settings and move "excel_creator.ExcelCreator.JSZip = oldJSZip;" to the "afterEach" method

**Review by jsdmitry — APPROVED** [review]

File: testing/helpers/jszipMock.js (hunk 2 above)
Comment (IgnatovDan) [reply testing/helpers/jszipMock.js:26]: There are different calls like 'folder/folder/file', 'file', 'folder/file':

- this._zip.folder("_" + RELATIONSHIP_PART_NAME).file("." + RELATIONSHIP_PART_NAME, relsFileContent);
- this._zip.file(CONTENTTYPES_FILE_NAME, this._getContentTypesContent());
- 'folder = this._zip.folder(XL_FOLDER_NAME),' + 'folder.folder("_" + RELATIONSHIP_PART_NAME).file(WORKBOOK_FILE_NAME + ".rels", relsXML);'

File: testing/helpers/jszipMock.js (hunk 1 above)
Comment (IgnatovDan) [reply testing/helpers/jszipMock.js:3]: There are 'this._zip.folder(XL_FOLDER_NAME).folder(WORKSHEETS_FOLDER).file(WORKSHEET_FILE_NAME, xmlResult.join(""));' entries and 'folders' can contain 'folders'.

**Pull request merged**